File: PlusMinus.cpp
void plusMinus(vector<int> arr) {
    int precision_level = 6;
    float po, ne, ze;
    po = ne = ze = 0;
    int n = arr.size();
    for(int i = 0; i < n; i++){
        if (arr[i] > 0)
            po++;
        else if (arr[i] < 0)
            ne ++;
        else ze++;
    }
    cout << fixed << setprecision(precision_level);
    cout << po/n << endl;
    cout << ne/n << endl;
    cout << ze/n << endl;
}

File: PlusMinus.py
def plusMinus(arr):
    positive = negative = zero = 0;
    
    n = len(arr)
    
    for i in range(n):
        if arr[i] > 0:
            positive += 1
        elif arr[i] < 0:
            negative += 1
        else:
            zero += 1
            
    print(f"{positive / n:.6f}")
    print(f"{negative / n:.6f}")
    print(f"{zero / n:.6f}")
